pkg/applicationserver/io/packages/alcsync/v1/data.go

```diff
@@ -0,0 +1,85 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alcsyncv1
+
+import (
+	"go.thethings.network/lorawan-stack/v3/pkg/errors"
+	"go.thethings.network/lorawan-stack/v3/pkg/ttnpb"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+var (
+	errFieldNotFound    = errors.DefineNotFound("field_not_found", "field `{field}` is not found")
+	errInvalidFieldType = errors.DefineCorruption("invalid_field_type", "field `{field}` has the wrong type `{type}`")
+)
+
+type packageData struct {
+	FPort            uint32
+	ThresholdSeconds uint32
```

Comment:
Use a `time.Duration` please, and use `time.Time.String()` / `time.Parse` in the marshaling.